client/src/app/site/pages/organization/pages/orga-meetings/pages/meeting-list/services/meeting-list-filter/meeting-list-filter.service.ts

```diff
@@ -0,0 +1,42 @@
+import { Injectable } from '@angular/core';
+import { TranslateService } from '@ngx-translate/core';
+import { BaseFilterListService } from 'src/app/site/base/base-filter.service';
+import { RelatedTime, ViewMeeting } from 'src/app/site/pages/meetings/view-models/view-meeting';
+import { ActiveFiltersService } from 'src/app/site/services/active-filters.service';
+import { OsFilter } from 'src/app/ui/modules/list';
+
+import { MeetingListServiceModule } from '../meeting-list-service.module';
+
+@Injectable({
+    providedIn: MeetingListServiceModule
+})
+export class MeetingListFilterService extends BaseFilterListService<ViewMeeting> {
+    protected storageKey = `MeetingList`;
+
+    public constructor(store: ActiveFiltersService, private translate: TranslateService) {
+        super(store);
+    }
+
+    protected getFilterDefinitions(): OsFilter<ViewMeeting>[] {
+        return [
+            {
+                property: `isArchived`,
+                label: this.translate.instant(`Archived`),
+                options: [
+                    { label: this.translate.instant(`Is Archived`), condition: true },
+                    { label: this.translate.instant(`Is not archived`), condition: [false, null] }
+                ]
+            },
+            {
+                property: `relatedTime`,
+                label: this.translate.instant(`Time`),
+                options: [
+                    { label: this.translate.instant(`Today`), condition: RelatedTime.Current },
+                    { label: this.translate.instant(`Ended`), condition: RelatedTime.Past },
+                    { label: this.translate.instant(`Future`), condition: RelatedTime.Future },
+                    { label: this.translate.instant(`Dateless`), condition: RelatedTime.Dateless }
```

Comment:
Labels and options are automatically translated by the component, replace all `instant` calls with the marker function